integration/containerd/runk/runk-tests.sh

```diff
@@ -63,6 +63,27 @@ test_runk() {
     # one line is the titles, and the other 2 lines are porcess info
     [ "3" == "$(sudo ctr t ps ${CONTAINER_ID} | wc -l)" ] || die "ps command failed"
 
+    echo "test pause and resume"
+    # The process outputs lines into /tmp/foo, which can be read in host when it's frozon.
+    sudo ctr t exec --detach --exec-id id2 ${CONTAINER_ID} sh -c "while true; do echo hello >> /tmp/foo; sleep 0.1; done"
```

Comment:
`/tmp/foo` can be replaced with `/tmp/${CONTAINER_ID}` or other variable to avoid hadcode at other place.